CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## unreleased
 
+* Updated all dependencies to the latest including Jest 29 and Typescript 5.
+* Updated node requirement to `^14.17.0 || ^16.13.0 || >=18.0.0`
```

Comment:
```suggestion
* Updated the minimum version of Node to `14.17.0`.
```

Reply:
I knew that some module drops node 17(ex: eslint-plugin-jest).
You can check the detailed list in WRP-11567.

Is it OK to describe the minimum version only?

Reply:
Updated the suggestion.